cumulusci/tasks/preflight/recordtypes.py

```diff
@@ -0,0 +1,15 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from collections import defaultdict
+
+
+class CheckSObjectRecordTypes(BaseSalesforceApiTask):
+    def _run_task(self):
+        rts = defaultdict(list)
+        records = self.tooling.query("Select SObjectType,FullName FROM RecordType")[
```

Comment:
```suggestion
        records = self.tooling.query("Select SobjectType, FullName FROM RecordType")[
```

Suggesting that we use the capitalization from the API docs (as below) because otherwise we have to remember that the schema's case-insensitive in the API and case-sensitive when represented in Python.

Reply:
Great reminder and important detail